tests/api_resources/test_file.py

```diff
@@ -28,7 +28,11 @@ def test_is_retrievable(self, request_mock):
         request_mock.assert_requested("get", "/v1/files/%s" % TEST_RESOURCE_ID)
         assert isinstance(resource, stripe.File)
 
-    def test_is_creatable(self, setup_upload_api_base, request_mock):
+    def test_is_creatable(self, mocker):
+        hc = mocker.Mock(stripe.http_client.HTTPClient)
```

Comment:
don't we have an existing fixture for this something like this?

Reply:
Not in master :'(